doc/cmd/flags/error-reporting.md

```diff
@@ -0,0 +1,31 @@
+# Error reporting
+
+Here is a list of data we retrieve when error reporting is enabled.
+
+* **date**: Event date
+* **os name**: Operating system (string, e.g. : "linux | mac | windows")
+* **architecture**: Your CPU kind (string, e.g. : "amd64 | i389")
+* **num_cpu**: Your cpu cound (int, e.g. : 8)
+* **release**: driftctl version (string, e.g. : "v0.2.2")
+* **server_name**: Your computer hostname (string, e.g. : "yourhostname")
+* **runtime version**: Golang version (string, e.g. : "go1.15.2")
+* **runtime infos**: Variables go_maxprocs, go_numcgocalls, go_numroutines
+* **packages**: Golang used packages and their versions
+* **stacktrace**: The error stack
+
+## Example of full output
```

Comment:
Either `Example of a full report output`, or `Real case example` or simply `Example`